### DISCUSSION TITLE
过期：驱动过时 性能不足

### DISCUSSION THREAD

**Discussion by cxing1198** [hooke007/mpv_PlayKit#301] [回收]
RIFE倍帧叠加一些Anime4K滤镜经常看着看着就闪退了，有时空降也会闪退，有啥办法解决吗，2070显卡

**Comment by hooke007** [comment]
性能不够，降低质量

**Reply by hooke007** [reply]
另外你说的是旧版的rife_2x? 2070应该用rife_cuda

**Reply by cxing1198** [reply]
嗯 rife_2x，之前一直用的标准版，rife_cuda怎么用啊😱，vsCuda包下下了然后怎么操作呢，覆盖进去也没用啊😭

**Reply by hooke007** [reply]
正常启用啊，没用是什么意思，控制台没消息还是stats帧率没变化

**Reply by cxing1198** [reply]
覆盖后启用那个RIFE_补桢_NVIDIA吗，帧率没一点变化，还是原视频帧率

**Reply by hooke007** [reply]
只有文字我不知道发生了什么 发控制台的图

**Reply by cxing1198** [reply]
![MPV-00_02_47 501-N0001](https://user-images.githubusercontent.com/82395219/233538348-433d5835-f59c-4b7e-8641-94612d8a25da.png)

**Reply by hooke007** [reply]
cuda driver balabala
你的显卡驱动很老吗
![Snipaste_2023-04-21_05-15-53](https://user-images.githubusercontent.com/41094733/233539596-e7f8a770-f321-4a7b-9742-d018729ed8dd.jpg)

**Reply by cxing1198** [reply]
是有点老，466.77版本，我去升级下

**Reply by cxing1198** [reply]
![GTRP~II}BMOQG$2VD@L$A}1](https://user-images.githubusercontent.com/82395219/233545741-39230c06-1957-4188-a269-68e83d8e8972.png)

**Reply by cxing1198** [reply]
可以用了，感谢巨佬
